arquillian/arquillian-tomee-tests/arquillian-tomee-jaxrs-tests/src/test/resources/spring-webmvc-pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.1.11</version>
```

Comment:
Spring Boot 3 has a Java 17 baseline, so this will likely fail the Java 11 based build. I think, that we would need to build an exclusion for that test if we run with a runtime below 17.